shpc/client/__init__.py

```diff
@@ -103,7 +103,21 @@ def get_parser():
     install.add_argument(
         "--symlink-tree",
         dest="symlink",
-        help="install to symlink tree too.",
+        help="install to symlink tree too (overrides settings.yml).",
+        default=None,
```

Comment:
Oh this is neat I’ve never seen it before! So if —symlink-tree is set, the None is True and this overrides the default False of no symlink tree (should that be Nome too)? And then if no symlink tree is True the None is ignored? What happens if both flags are provided?

Reply:
Perhaps we should use store_const here to make the three cases more clear? https://stackoverflow.com/a/34750557

Reply:
> this overrides the default False of no symlink tree (should that be None too)

It works as it is: None if no option given, and True / False for `--symlink-tree` / `--no-symlink-tree`

> What happens if both flags are provided?

If both are given, the last one wins

> Perhaps we should use store_const here to make the three cases more clear? https://stackoverflow.com/a/34750557

A matter of choice :) I find `store_true` and `store_false` very explicit :)

Reply:
Ok, I’m cool with that :)

will probably finish review tomorrow - I want to look everything over once more with a clear head, and I’m rather tired and still a bit on edge from the day. So stay tuned for tomorrow! I think the default version PR looks great (and can update the PR here) so probably I’ll merge that one first.

goodnight from the inferno! lol

![F1103B3A-C907-4872-91DA-C70F9447F509](https://user-images.githubusercontent.com/814322/160266558-202c9c55-2bdc-4e99-afdc-8a55fc20b96d.jpeg)